src/api/token-bridge/create-token-bridge-api.js

```diff
@@ -0,0 +1,209 @@
+import Web3 from "web3";
+import KlerosLiquidExtraViews from "../../assets/contracts/kleros-liquid-extra-views.json";
+import TokenBridgeEthereum from "../../assets/contracts/token-bridge-ethereum.json";
+import TokenBridgeXDai from "../../assets/contracts/token-bridge-xdai.json";
+import WrappedPinakion from "../../assets/contracts/wrapped-pinakion.json";
+import Pinakion from "../../assets/contracts/pinakion.json";
+import { getBaseUrl } from "../../helpers/block-explorer";
+import * as ethereum from "./ethereum-api";
+import * as xDai from "./xdai-api";
+
+const { toHex } = Web3.utils;
+
+export const supportedChains = {
+  // Mainnet
+  // 1: {
+  //   chainId: 1,
+  //   chainName: "Mainnet",
+  //   nativeCurrency: { name: "ETH", symbol: "ETH", decimals: 18 },
+  //   rpcUrls: [ensureEnv("REACT_APP_WEB3_FALLBACK_URL")],
+  //   blockExplorerUrls: [getBaseUrl(1)],
+  //   side: "ethereum",
+  //   counterPartyChainId: 100,
+  // },
+  // Kovan
+  42: {
+    chainId: 42,
+    chainName: "Kovan",
+    nativeCurrency: { name: "ETH", symbol: "ETH", decimals: 18 },
+    rpcUrls: [ensureEnv("REACT_APP_WEB3_FALLBACK_KOVAN_URL")],
+    blockExplorerUrls: [getBaseUrl(42)],
+    side: "ethereum",
+    counterPartyChainId: 77,
+  },
+  // xDai
+  // 100: {
+  //   chainId: 100,
+  //   chainName: "xDAI",
+  //   nativeCurrency: { name: "xDAI", symbol: "xDAI", decimals: 18 },
+  //   rpcUrls: [ensureEnv("REACT_APP_WEB3_FALLBACK_XDAI_URL")],
+  //   blockExplorerUrls: [getBaseUrl(100)],
+  //   side: "xDai",
+  //   counterPartyChainId: 1,
+  // },
+  // Sokol
+  77: {
+    chainId: 77,
+    chainName: "Sokol",
+    nativeCurrency: { name: "Sokol POA", symbol: "SPOA", decimals: 18 },
+    rpcUrls: [ensureEnv("REACT_APP_WEB3_FALLBACK_SOKOL_URL")],
+    blockExplorerUrls: [getBaseUrl(77)],
+    side: "xDai",
+    counterPartyChainId: 42,
+  },
+};
+
+export default async function createTokenBridgeApi(provider) {
+  const web3 = new Web3(provider);
+  // Accounts is ignored because it's purpose is only to check whether
+  // the provider can sign transactions or not.
```

Comment:
Yeah, it will throw, then the API won't be initialized.
Currently the error bubbles up to the UI... This shouldn't happen, it's just here because I'm a bit paranoid.